python/ray/tune/suggest/skopt.py

```diff
@@ -121,6 +125,7 @@ def __init__(self,
                  mode: Optional[str] = None,
                  points_to_evaluate: Optional[List[Dict]] = None,
                  evaluated_rewards: Optional[List] = None,
+                 convert_to_python: bool = False,
```

Comment:
Do we plan to set this to true?

Reply:
I left this at `False` to not impede backwards compatibility. It's probably a very minor change though, so we could just enable this per default.

It seems I have to fix one more test anyway, so I'll change it there.